src/sentry/conf/server.py

```diff
@@ -3950,6 +3950,9 @@ def custom_parameter_sort(parameter: dict) -> tuple[str, int]:
 # Disable DDM entirely
 SENTRY_DDM_DISABLE = os.getenv("SENTRY_DDM_DISABLE", "0") in ("1", "true", "True")
 
+SEER_SIMILARITY_MODEL_VERSION = "0"
```

Comment:
would this not need to be `v0`?

Reply:
Yes. Damn - this is what I get for trying to juggle so many branches at once. I did fix that... just apparently not in this branch. 🤦🏻‍♀️

Good catch.